packages/@tailwindcss-vite/src/index.ts

```diff
@@ -95,10 +95,18 @@ export default function tailwindcss(): Plugin[] {
       candidates.add(candidate)
     }
 
+    // Watch individual files
     for (let file of result.files) {
       addWatchFile(file)
     }
 
+    // Watch globs
+    for (let glob of result.globs) {
+      if (glob.glob[0] === '!') continue
+
+      addWatchFile(`${glob.base}/${glob.glob}`)
```

Comment:
Note from our call: We need to be careful about this on Windows since we want glob to always use posix style separators but base can use windows-style. We will need to fix windows path issues anyways and will take care of this as part of https://github.com/tailwindlabs/tailwindcss/pull/14065